### PR TITLE
Allow users to view their own SSH keys

### LINKED ISSUE #123: `whoami` command should be able to list your own keys too
Currently we can run `whoami` to verify your user against a lagoon. But it would be good to be also able to view your own keys.

This feature request is to be able to output your keys too.

### PR DESCRIPTION
**Pull request by shreddedbacon**
This allows users to view their own SSH keys using the whoami command

closes #123

### CODE REVIEW

**Comment by shreddedbacon** [comment]
@smlx just want your eyes over this as it changes the SSHKey struct, which means more information is exported and imported for users. 

I don't think it will be a problem though.

File: internal/schema/sshKey.go [hunk 1]
```diff
@@ -4,9 +4,11 @@ import "github.com/amazeeio/lagoon-cli/pkg/api"
 
 // SSHKey is the basic SSH key information, used by both config and API data.
 type SSHKey struct {
-	Name     string         `json:"name"`
-	KeyValue string         `json:"keyValue"`
-	KeyType  api.SSHKeyType `json:"keyType"`
+	Name           string         `json:"name"`
+	KeyValue       string         `json:"keyValue"`
+	Created        string         `json:"created"`
```
Comment (smlx) [inline internal/schema/sshKey.go:9]: I was going to say that the CLI output doesn't have a timezone so maybe you could unmarshal to a `time.Time` here, but looking at the debug output the _response_ doesn't have a timezone either :cry: 

I'll raise a lagoon issue for that.

File: cmd/whoami.go [hunk 2]
```diff
@@ -43,23 +54,55 @@ This is useful if you have multiple keys or accounts in multiple lagoons and nee
...
-					returnNonEmptyString(fmt.Sprintf("%v", user.ID)),
-					returnNonEmptyString(user.Email),
-					returnNonEmptyString(user.FirstName),
-					returnNonEmptyString(user.LastName),
+		if showKeys || showFingerprints {
+			// if we are only showing the users keys, leave the users email visible as part
+			// of helping identify the user stil
+			var keys []output.Data
+			for _, key := range user.SSHKeys {
+				if showFingerprints {
```
[the remark this answers is not in the document]
Comment (shreddedbacon) [reply cmd/whoami.go:62]: Yeah, I thought that but it is a lot of information on screen when its an RSA key

File: internal/schema/sshKey.go (hunk 1 above)
Comment (shreddedbacon) [reply internal/schema/sshKey.go:9]: We might be able to just assume UTC in this case even, as I think all API pods run in UTC typically.

File: cmd/whoami.go (hunk 2 above)
[the remark this answers is not in the document]
Comment (shreddedbacon) [reply cmd/whoami.go:62]: I'll see if I can make it display nicer

**Pull request merged**